easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.8-iimpi-2020a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'GlobalArrays'
+version = '5.8'
+
+homepage = 'https://hpc.pnl.gov/globalarrays'
+description = "Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model"
+
+toolchain = {'name': 'iimpi', 'version': '2020a'}
+
+source_urls = ['https://github.com/GlobalArrays/ga/releases/download/']
+sources = ['v%(version)s/ga-%(version)s.tar.gz']
+checksums = ['64df7d1ea4053d24d84ca361e67a6f51c7b17ed7d626cb18a9fbc759f4a078ac']
+
+dependencies = [
+    ('imkl', '2020.1.217'),
```

Comment:
@nordmoen This basically means the `intel` toolchain should be used instead, no?

Reply:
The reason I did it this way was because I saw `HDF5` compiled with `iimpi`, can I mix `iimpi` and `intel`?

Reply:
As long as the version is the same, yes you can "mix" them.

`iimpi` is a subtoolchain of `intel`, and EasyBuild considers dependencies installed with a subtoolchain, so it'll use the HDF5 installed with `iimpi` to resolve that dep, yes.

If you change `GlobalArrays` to use `intel` as toolchain, you'll need to do the same for `OpenMolcas` of course (note that this involves renaming the easyconfig files too).

Reply:
Moved both back to intel